packages/storybook/stories/components/datepicker/datepicker.web-components.stories.ts

```diff
@@ -38,6 +44,16 @@ const storyParams = {
     category: 'General',
     defaultValue: false,
   },
+  maxDate: {
+    category: 'General',
+    defaultValue: null,
+    options: [null, new Date('1999-11-02'), new Date('2024-11-02')],
+  },
+  minDate: {
```

Comment:
I have an error on storybook with the date
![image](https://github.com/ovh/design-system/assets/27487948/ffd686e3-eb4e-4490-bd9e-653e3e0710b2)
And when I select one the input not display the value